web/src/components/home/subcomponents/what_is_wcs/what_is_wcs.js

```diff
@@ -1,5 +1,42 @@
+import { Box, Stack, Typography } from "@mui/material";
+import whatIsWcsStyles from "./what_is_wcs.styles";
+import messages from "./messages";
+
 function WhatIsWcs() {
-  return <div>What is west coast swing</div>;
+  return (
+    <Box sx={whatIsWcsStyles.whatIsWcsStylesContainer}>
+      <Typography variant="h4" sx={whatIsWcsStyles.header}>
+        What is West Coast Swing?
+      </Typography>
+      <Box sx={whatIsWcsStyles.itemsContainer}>
+        <Stack sx={whatIsWcsStyles.twoColumnContent}>
+          <Box sx={whatIsWcsStyles.videoContainer}>
+            <iframe
+              width="100%"
+              height="100%"
+              src="https://www.youtube.com/embed/v4fRV0aG3lc?si=BKXYaAPC8ozlSUWF"
+              title="YouTube video player"
+              frameborder="0"
+              allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+              referrerpolicy="strict-origin-when-cross-origin"
+              allowfullscreen
+            ></iframe>
+          </Box>
+          <Stack
+            sx={whatIsWcsStyles.descriptionContainer}
+            spacing={{ xs: 1, md: 2 }}
```

Comment:
Nit: I've been moving stuff like this into the styles object too

Reply:
I couldn't get this specific property into the styles object, it wouldn't translate to an actual css property for some reason